lib/uksched/include/uk/sched.h

```diff
@@ -72,6 +72,9 @@ typedef void  (*uk_sched_thread_blocked_func_t)
 typedef void  (*uk_sched_thread_woken_func_t)
 		(struct uk_sched *s, struct uk_thread *t);
 
+typedef const struct uk_thread * (*uk_sched_idle_thread_t)
```

Comment:
`uk_sched_idle_thread_func_t` would be more clear, and more in compliance to our guidelines, as the current name implies "thread type" rather than "function type".

Reply:
You are totally right 👍🏻